app/decorators/account_decorator.rb

```diff
@@ -3,8 +3,24 @@
 class AccountDecorator < ApplicationDecorator
   delegate :display_name, :email, to: :admin_user, prefix: true
 
+  def first_admin
+    @first_admin ||= (first_admin_cached || super)
+  end
```

Comment:
```suggestion
  def first_admin
    @first_admin ||= users.loaded? ? users.find { |user| user.admin? && user.impersonation_admin? }  : super
  end
```

Where `user.impersonation_admin?` needs to be implemented in `User`.

Reply:
I used this code, and I also esu the `:admin_users` association instead of `:users`, since `first_admin` only searches among admins.